seed/challenges/02-javascript-algorithms-and-data-structures/debugging.json

```diff
@@ -68,6 +68,44 @@
       "translations": {}
     },
     {
+      "id": "587d7b83367417b2b2512b37",
+      "title": "Understanding the Differences between the freeCodeCamp and Browser Console",
+      "description": [
+        "You may have noticed that some freeCodeCamp JavaScript challenges include their own console.  This console behaves a little differently than the browser console you used in the last challenge.",
+        "The following challenge is meant to highlight some of the differences between the freeCodeCamp console and the browser console.",
+        "First, the browser console.  When you load and run an ordinary JavaScript file in your browser the <code>console.log()</code> statements will print exactly what you tell them to print to the browser console the exact number of times you requested. In your in-browser text editor the process is slightly different and can be confusing at first.",
+        "Values passed to a <code>console.log</code> in the text editor block run each set of tests as well as one more time for any function calls that you have in you code, i.e. <code>returnSum();</code>",
+        "This lends itself to some interesting behavior and might trip you up in the beginning, because a logged value that you expect to see only once may print out many more times depending on the number of tests and the values being passed to those tests.",
+        "If you would like to see only your single output and not have to worry about running through the test cycles, you can use <code>console.clear()</code>.",
+        "The fCC console on the other hand is used for feedback while passing the test cases in fCC and does not print your <code>console.logs()</code>",
+        "<hr>",
+        "Use <code>console.log()</code> to print the variables in the code where indicated.",
+        "",
+        "Interested in learning more: Try figuring out how to print only one variable. Experiment with <code>console.clear()</code> in between your <code>console.logs()</code> and make a note of which statement prints."
+      ],
+      "challengeSeed": [
+        "// Open your browser console",
+        "let outputTwo = \"This will print to the browser console 2 times\";",
+        "// Use console.log() to print the outputTwo variable",
+        "let outputFour = \"This will print to the browser console 4 times, twice for the logs times two tests\";",
+        "// Use console.log() to print the outputFour variable",
+        "",
+        "",
+        "",
+        ""
+      ],
+      "tests": [
+        "assert(code.match(/console\\.log\\(outputTwice\\)/g), 'message: Use <code>console.log()</code> to print the <code>outputTwice</code> variable.  In your Browser Console this should print out the value of the variable two times.');",
```

Comment:
Just fix the regex so it tests for `outputTwo` instead of `outputTwice`, and the message to update for the new variable name. Also, when I ran this locally (on Firefox), the browser console printed both statements 2x - it didn't print the `outputFour` one 4x. Is `outputFour` supposed to be in a function call?

Reply:
hmm...  maybe a difference in firefox and chrome, I'll add it to the list

Reply:
@ed42311 - most of fCC's user base uses Chrome (I don't have it on my machine, though) - if the behavior is as-expected for Chrome, then leave the challenge as-is, and add a note about seeing different results based on browser. 👍